stock-tracker/src/utils/payload.ts

```diff
@@ -2,5 +2,5 @@ import { AppState } from "../store/rootReducer";
 
 export type Payload = {
```

Comment:
This could probably be better named. Something like `ApiPayload` or `ServicePayload` would be better, even.